pkg/util/predicates.go

```diff
@@ -71,6 +73,49 @@ func (p FinalizersChangedPredicate) Update(e event.UpdateEvent) bool {
 		!reflect.DeepEqual(e.MetaOld.GetFinalizers(), e.MetaNew.GetFinalizers())
 }
 
+// FilterForOwner will only allow events that owned by noobaa
+type FilterForOwner struct {
+	OwnerSubstrings []string
```

Comment:
Why do you need an array where the list can implicitly be created by stacking more than one predicate on the watch? In most cases, you just use a single string. 
As an added bonus, moving to a single string will make the code simpler